react/features/video-layout/functions.js

```diff
@@ -74,5 +74,7 @@ export function shouldDisplayTileView(state: Object = {}) {
         state['features/video-layout']
             && state['features/video-layout'].tileViewEnabled
             && !state['features/etherpad'].editing
+            && (typeof interfaceConfig === 'undefined'
```

Comment:
This basically means that if interface config is undefined we default to tile view. Is there any reason to do that?

Reply:
I added the check for mobile. Turns out though state['features/etherpad'] might also need a truthy check.